packages/react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -223,12 +228,16 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
           [`.${classNames.buttonLabel}`]: fonts.medium,
           ':hover': {
             backgroundColor: palette.themeLighter,
-            borderColor: palette.themeLighter,
             color: palette.themePrimary,
+            border: `2px solid ${palette.black}`,
+            outline: `1px solid ${palette.white}`,
+            outlineOffset: '-3px',
           },
           ':focus': {
             backgroundColor: palette.themeLighter,
-            borderColor: palette.white,
+            border: `2px solid ${palette.black}`,
```

Comment:
This is my bad, the outer border in v7/8 is actually supposed to be 1px, and it'll only be 2px in vNext.

Reply:
I've changed the border size to 1px and also reduced the outline offset from -3px to -2px just because it looks better :)